views/index.erb

```diff
@@ -47,13 +266,62 @@
 
 <!-- Add your Javascript to this script tag -->
 <script language="javascript">
-  console.log("I am Javascript!");
 
-  var profile_values = document.getElementsByClassName('profile-value');
-  var total_words = 0;
+  $(document).ready(function(){
+    var profile_section = $.trim($(".profile").text()).length;
+      console.log(profile_section);
+    var code_section = $.trim($(".other").text()).length;
+      console.log(code_section);
+    var total_words = profile_section + code_section;
+      console.log(total_words);
+    $(".words").html(total_words);
+  });
+
+</script>
+
+<script type="text/javascript">
+
+  $(document).ready(function(){
+
+  $('.icon').hover(
```

Comment:
See also CSS `:hover` pseudo classes, which are preferable for visual tricks like this...